es-app/src/SystemData.h

```diff
@@ -8,23 +8,31 @@
 #include "PlatformId.h"
 #include "ThemeData.h"
 #include "FileFilterIndex.h"
+#include "CollectionSystemManager.h"
+
+struct SystemEnvironmentData
```

Comment:
Not at all, thanks for catching that. The compiler on the Pi doesn't warn me of anything!